geode-core/src/distributedTest/java/org/apache/geode/cache/client/ClientServerRegisterInterestsDUnitTest.java

```diff
@@ -19,10 +19,9 @@
 import static org.apache.geode.distributed.ConfigurationProperties.LOG_FILE;
 import static org.apache.geode.distributed.ConfigurationProperties.LOG_LEVEL;
 import static org.apache.geode.distributed.ConfigurationProperties.MCAST_PORT;
-import static org.junit.Assert.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.assertFalse;
```

Comment:
Since you are in here can we constantly replace all these with AssertJ. There is a handle plug-in for IntelliJ for this.

Reply:
What plugin is that?

Reply:
Got it now.